template-sidebar.php

```diff
@@ -1,9 +1,15 @@
 <?php
 /**
- * The template for displaying full width pages.
+ * Template for displaying pages with a sidebar.
  *
- * Template Name: Full width
+ * In Storefront 2.5.7 and earlier, the default page template included a
+ * sidebar. In 2.5.8 the sidebar was removed from the default template.
+ * This template was added so stores can opt-in to sidebar layout on a
+ * per-page basis.
  *
+ * Template Name: Sidebar
```

Comment:
Agreed, this could be confusing. Let's go with `Page with sidebar` - then it looks like this in the editor:

<img width="388" alt="Screen Shot 2020-05-27 at 10 39 57 AM" src="https://user-images.githubusercontent.com/4167300/82957132-6e5c1d80-a006-11ea-818c-822cd1a39191.png">

2ef6e480c5279f77951a5415e42abd6d9864d433

Reply:
While we're looking at template names, I wondered about giving the default template a name, so it's clearer that it's full width. It shows up twice in the dropdown, but maybe this is more clear:

<img width="388" alt="Screen Shot 2020-05-27 at 10 39 57 AM" src="https://user-images.githubusercontent.com/4167300/82957576-6badf800-a007-11ea-8e07-777679aee6d7.png">

The user's menu selection sticks - `Default template` clears out the template meta field, `Full-width page` sets it to `page.php` (i.e. default template). @Aljullu what do you think?

```sh
%  docker-compose exec wordpress wp --allow-root post meta get 206 _wp_page_template    
%  docker-compose exec wordpress wp --allow-root post meta get 206 _wp_page_template
page.php
```

Reply:
I'm thinking it's unusual to add a name to the default template - probably less confusing to leave it as just `Default template`.

Reply:
> I'm thinking it's unusual to add a name to the default template - probably less confusing to leave it as just `Default template`.

Agree with this. :100: